sdk/appconfiguration/packages.md

```diff
@@ -0,0 +1,9 @@
+
+# Package Index - appconfiguration
+
+| Package Id     | Readme    | Changelog                 | Published Url       |
+|----------------|-----------|---------------------------|---------------------|
+| [`azure-appconfiguration`]( azure-appconfiguration/setup.py )| [README](azure-appconfiguration/README.md) |  N/A  | [PyPI]( https://pypi.org/project/azure-appconfiguration ) |
```

Comment:
Accidentally added? Probably should be removed right?

Reply:
Yup. I thought I deleted before pushing. Will fix it and repush.